### ISSUE TITLE
Missing lots of spells

### ISSUE THREAD

**Issue by jpcguy89** [tadzik/5e-spells#1]
I'm working on completing it now. Will send or push updated file once complete. Also, did you mark if the spell could be cast as a ritual in the school fields (e.g. "transmutation (ritual)")?

**Comment by matt-richard** [comment]
Did you ever finish adding the rest of the spells? I'd love to help, a complete list of spells in JSON format would be extremely helpful

**Comment by jpcguy89** [comment]
Yes I did. Just got wrapped up in another project. Here they are. My apologies. Please note, that there are typos and they need to be gone through with a fine tooth comb.

[Archive.zip](https://github.com/tadzik/5e-spells/files/635011/Archive.zip)

**Comment by jpcguy89** [comment]
Also, I may have in a few instances swapped two values by mistake. Just keep an eye out for anything that doesn't seem right.
